atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -1079,6 +1079,23 @@ private void testGetRangeWithTimestamps(boolean reverse) {
     }
 
     @Test
+    public void testGetRangeOfTimestampsReturnsAllRows() {
+        keyValueService.put(TEST_TABLE,
+            ImmutableMap.of(
+                Cell.create(row0, column0), value0_t0,
+                Cell.create(row1, column0), value0_t0,
+                Cell.create(row2, column0), value0_t0),
+            TEST_TIMESTAMP);
```

Comment:
For the most part I think so, though we do exclude a couple of rules - @gsheasby would probably know this better!